### DISCUSSION TITLE
Inference

### DISCUSSION THREAD

**Discussion by hundredeuk2** [BoocampMonster/korean-relation-extraction#22] [Q&A] [answered]
https://github.com/BoocampMonster/korean-relation-extraction/blob/6c6846d22d044d8c3a3ed7897fbbbc3af55ddc93/inference.py#L84

여기 부분 원래 포맷이 이런거야?

**Comment by yhkee0404** [comment, marked as answer]
이랬던 거 같음: 0cf71a2b094124f2af5e5e0470c04420bcc4b934
https://github.com/BoocampMonster/korean-relation-extraction/blob/main/configs/baseline.yaml#L13

**Reply by hundredeuk2** [reply]
오 감사합니다.

**Comment by yhkee0404** [comment]
#25
